in-market-test/v2/README.md

```diff
@@ -0,0 +1,110 @@
+# In-Market Test (2024) - Helper Party Runbook
+
+The runbook is set up to use [ansible](https://docs.ansible.com/ansible/latest/) to provision the remote machine. You can, of course, decide to run the commands in the ansible scripts manually if you prefer.
+
+## Local Install
+
+First, install [pipx](https://github.com/pypa/pipx?tab=readme-ov-file#install-pipx). On a Mac
+
+```
+brew install pipx
+```
+
+Then install ansible
+
+```
+pipx install --include-deps ansible
+```
+
+
+## Provisioning
+
+
+### Launching
+You'll need to provision a host on your cloud provider with roughly the following specs:
+1. 32 vCPU
+2. 64 GB memory
+3. 12.5 Gbps network bandwidth
+
+On AWS, we use c6id.8xlarge with the Amazon Linux 2023 AMI.
+
+The host needs SSH access (from you, if you want to limit the IP range), and HTTPS access from the internet (e.g., port 443 needs to be open.) No other ports need to be opened.
+
+
+### SSH access
+
+You'll need to configure the ansible inventory and your ssh config. Save the SSH key provided by the cloud provider to your ssh directory, e.g. `~/.ssh/helper_connect.pem`, then update `~/.ssh/config` to include:
+
+```
+Host ipa-helper
+  HostName <ipa-helper-ip-or-public-dns-name>
+  User <associated-user>
+  IdentityFile ~/.ssh/helper_connect.pem
+```
+
+Now, copy the inventory template
+
+```
+cp in-market-test/v2/ansible/inventory-template.ini in-market-test/v2/ansible/inventory.ini
+```
+and update it with the host name used in the SSH config and the assigned identity (1,2, or 3):
+
+```
+[myhosts]
+ipa-helper           identity=1
+
+[myhosts:vars]
+ansible_python_interpreter=/usr/bin/python3
+```
+
+### Install dependencies and compile IPA
+
+This command installs dependencies and compiles IPA. Note: it uses `yum`. If you need to use a different package manager, you can update the file.
+```
+ansible-playbook -i in-market-test/v2/ansible/inventory.ini in-market-test/v2/ansible/provision.yaml
```

Comment:
is there a way to instruct ansible to use the default value for `-i`? if no, maybe setting an environment variable that points to the runbook makes this instruction easier to maintain

Reply:
[default value](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html) is `/etc/ansible/hosts`, and I'd prefer to keep it project specific. given most of these are just going to be copy and pasted, an environmental variable just seems like more room for something to get messed up. with this as is, you can copy and paste it without the need for anything else.

Reply:
i'm going to keep this as is, but if you feel strongly let's sync offline